pkg/server/server.go

```diff
@@ -273,6 +274,7 @@ func New(
 
 	kineConfig.Listener = listen
 	kineConfig.Endpoint = fmt.Sprintf("dqlite://k8s?%s", params.Encode())
+	kineConfig.PollAfterTimeout = pollAfterTimeout
```

Comment:
I think we should update to use our [existing approach](https://github.com/canonical/k8s-dqlite/blob/ca5713a2f267ef2a8b30f165f7675b36cec24ba0/pkg/server/server.go#L264-L278) of passing parameters through the driver string.